<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RustDaVinci</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/RustDaVinci-icon.ico</normaloff>../resources/RustDaVinci-icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadImagePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>               Load Image...             </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/load_image_icon.png</normaloff>../resources/load_image_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="identifyAreasPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>    Identify palette and frame...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/select_area_icon.png</normaloff>../resources/select_area_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="paintImagePushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>               Paint Image...            </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/paint_image_icon.png</normaloff>../resources/paint_image_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>                 Settings                    </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/settings_icon.png</normaloff>../resources/settings_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
